thirtyinch-lint/README.md

```diff
@@ -0,0 +1 @@
+# ThirtyInch - Lint
```

Comment:
No one of our modules have its own README.
Even if this brings no value I prefer to remove it